### PR TITLE
Ui admin

### PR DESCRIPTION
**Pull request by svelasco7**
adding stats for an admin to view; has user count, messages count,  conversation count, & newest user. also has template for uploading file but does not currently do anything

### CODE REVIEW

**Review by dcollier — COMMENTED** [review]
Looks good overall - the servlet structure makes sense :+1: . A few high level points:

1. Seeing as we don't have the ability to indicate if someone is an admin, I think we should make sure that the admin page link is visible from all pages (we'll need to change that later)

2. Running this code with the google-java-format tool will take care of the various whitespace issues

3. Adding tests? :)

**Comment by daphliu-zz** [comment]
The logic makes sense overall! I agree with Desiye to add some tests in. 

1. Small thing, but perhaps split "choose file", "no file chosen" and "submit" to separate lines?

2. The example provided had a dropdown of Romeo & Juliet scripts and I'm wondering why you're using "Choose file" instead? 

capture

**Comment by svelasco7** [comment]
yeah this is still just outline & testing; still need to only allow the view when we can determine if a user is an admin since I think it's still being worked on.; I think we would have to load in the data either way & having a preloaded options would reduce user error but i was thinking of using file upload to just give more freedom but I can always change it after I figure out how to set up the database for this

**Comment by daphliu-zz** [comment]
Also instead of AdminStats, I was thinking about using "AdminServlet" just to standardize it across the different names?

File: src/main/java/codeu/controller/testing.txt [hunk 1]
```diff
@@ -0,0 +1,124 @@
+ACT_I
```
Comment (dcollier) [inline src/main/java/codeu/controller/testing.txt]: Let's not include this file in this PR.

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 2]
```diff
@@ -90,7 +106,7 @@ public PersistentDataStore() {
     List<Conversation> conversations = new ArrayList<>();
 
     // Retrieve all conversations from the datastore.
-    Query query = new Query("chat-conversations");
+    Query query = new Query("chat-conversations").addSort("creation_time");
```
Comment (dcollier) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java]: Is there a reason for sorting by creation time here? (can we add a comment to indicate why?)

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 3]
```diff
@@ -46,6 +50,18 @@ public PersistentDataStore() {
     datastore = DatastoreServiceFactory.getDatastoreService();
   }
 
+  /*Retrieves a User Object given User UUID as string*/
```
Comment (dcollier) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java:52]: comment should be moved over one space

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 4]
```diff
@@ -79,7 +97,7 @@ public PersistentDataStore() {
     return users;
   }
 
-  /**
+  /**ß
```
Comment (dcollier) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java]: that's an interesting character?

**Review by dcollier — APPROVED** [review]
couple minor comments to look at - we can look at adding a bit more tests later :)

**Pull request merged**